src/index.html

```diff
@@ -3,11 +3,261 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>The MET</title>
+    <link
+      rel="shortcut icon"
+      href="./images/photos/THEMET.png"
+      type="image/png">
+    <!-- <link rel="stylesheet" href="./styles/main.scss"> -->
+    <link rel="stylesheet" href="/dist/css/main.css">
     <script src="scripts/main.js" defer></script>
   </head>
   <body>
-    <h1>Miami</h1>
+    <h1 hidden>The MET</h1>
+    <!-- <div class="container"> -->
+      <div class="met__bar" id="menu">
+        <div class="met__nav">
+          <div class="met__logo">
+            <img src="./images/photos/THEMET.png" alt="logo">
+          </div>
+          <a href="#" class="met__cancel">
+            <img src="./images/cross.svg" alt="cross">
+          </a>
+        </div>
+        <nav class="nav">
+          <ul class="nav-list">
+            <li class="nav-list__item">
+              <a href="/" class="nav-list__link">
+                Home
+              </a>
+            </li>
+
+            <li class="nav-list__item">
+              <a href="#about" class="nav-list__link">
+                About Us
+              </a>
+            </li>
+
+            <li class="nav-list__item">
+              <a href="#gallery" class="nav-list__link">
+                Gallery
+              </a>
+            </li>
+
+            <li class="nav-list__item">
+              <a href="#testimonials" class="nav-list__link">
+                Testimonials
+              </a>
+            </li>
+
+            <li class="nav-list__item">
+              <a href="#contact" class="nav-list__link">
+                Contact Me
+              </a>
+            </li>
+          </ul>
+        </nav>
+        <div class="met__info">
+          <div class="met__avenue">
+            <p class="avenue fifth__avenue">The Met Fifth Avenue</p>
+            <p class="avenue fifth__avenue">+1 212-535-7710</p>
+          </div>
+          <div class="met__cloister">
+            <p class="title cloister__title">The Met Cloisters</p>
+            <p class="mobile cloister__mobile">+1 212-923-3700</p>
+          </div>
+          <div class="call__order">
+            <a href="+1 212-535-7710">call to order</a>
+          </div>
+        </div>
+      </div>
+      <header class="header">
+        <div class="header__wrapper">
+          <div class="top header__top">
+            <img src="./images/photos/THEMET.png" class="logo top__logo" alt="The Met">
+            <div class="info top__info">
+              <img src="./images/phone.svg" class="info__phone" alt="phone">
```

Comment:
i see it in your code, but the deployed page doesn't have it
![Snapshot 2022-04-13 11 34 45](https://user-images.githubusercontent.com/39308717/163135352-47d875da-aa7c-483d-af0e-93fb8f6fddd7.png)